basis-universal/src/transcoding/enums.rs

```diff
@@ -447,7 +465,18 @@ impl TranscoderBlockFormat {
 
     /// Returns true if the block format is a compressed format.
     pub fn is_compressed(self) -> bool {
-        unsafe { !sys::basis_block_format_is_uncompressed(self.into()) }
```

Comment:
I'd rather we call the C implementation that try to duplicate it for same reasons as in previous comment